packages/bdd-tests/util/helper.ts

```diff
@@ -1,67 +1,73 @@
 import * as crypto from "crypto"
```

Comment:
This file probably wants carving up into some more descriptive ones. A job for another time though.

Reply:
yeah - because it originally used local variables to manage state it all needed to be in one file.  As it uses context to store state now it can be in multiple files
Have added a comment saying it should be split to smaller files